docs/manual/performance/http-caching.en.md

```diff
@@ -273,36 +399,65 @@ X-Cache-Tags: contao.db.tl_page.18, contao.db.tl_layout.16, contao.db.tl_content
+By the way: Invalidating cache entries also works via HTTP requests because the cache proxy does not necessarily
+have to be on the same server as Contao. Most cache proxies support receiving requests `PURGE` (yes, you are allowed
+to invent your own HTTP methods in addition to the usual ones such as `GET`, `POST` etc.) and the invalidation logic
+is completely abstracted in Contao so that you can connect any cache proxy programmatically if needed.
+This way, any cache proxy can be tought to support the cache tag based invalidation logic of Contao!
+{{% /notice %}}
 
 ## FAQ
 
-{{% expand "Welche Cache-Einstellungen sind für mich die »richtigen«?" %}}
-%It is not possible to answer this question in general, because as is so often the case in our industry, the answer is "it depends". However, there are a few basic principles that you can use as a guide:
-
-For example, it will rarely make sense to set the private cache duration higher than the shared cache duration, and if a page is only supposed to be in the shared cache for one hour, you seem to generally think that the visitor should see the new content after one hour at the latest. Therefore it makes no sense to configure the cache duration of the private cache to two hours. A rule of thumb could be: The duration of the shared cache should be equal or higher than the private cache.
-
-The choice of cache duration depends on how often content is changed on the page. A page with the imprint, for example, will almost certainly change very rarely. So we could generally leave the content in the cache longer. But maybe it is important to us that the visitor - if changes have been made - sees the new data quite quickly. Then we would accept that the client sends a request more often (= lower, private cache time) and leave our shared cache time relatively high. Thanks to Contao's cache tagging framework, which has already been explained, the cache would be emptied automatically when the content is changed or we could also have it emptied via maintenance mode. So we would generally choose lower cache times.
-{{% /expand %}}
-
-{{% expand "Heisst das, die Remember-Me-Funktion deaktiviert den Cache für einen Besucher komplett?" %}}
-%Yes. The function of Remember-Me is to automatically log a visitor in at the call of any URL, if this is not yet the case. It is therefore impossible to deliver the answer from the cache, otherwise a visitor would never be logged in. 
+{{% expand "What cache settings are the »right ones« for me?" %}}
```

Comment:
```suggestion
{{% expand "What cache settings are the "right ones" for me?" %}}
```

In the English version we should use regular quotes. At least that's also what DeepL automatically did elsewhere ;)

Reply:
```suggestion
{{% expand "What cache settings are the \"right ones\" for me?" %}}
```

Right, but they have to be escaped ;)

Reply:
Right 😬